MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.cxx

```diff
@@ -329,8 +360,20 @@ const char* vtkMRMLMarkupsToModelNode::GetInterpolationTypeAsString( int id )
   {
   case Linear: return "linear";
   case CardinalSpline: return "cardinalSpline";
-  case HermiteSpline: return "hermiteSpline";
   case KochanekSpline: return "kochanekSpline";
```

Comment:
That's also a reason why you want to store enums as strings in a file!!!
kochanekSpline has just become 2, while it was 3 before. If you stored "3" in a file as interpolation type, user would be surprised to see that when he loads an earlier file, interpolation type is changed to polynomial.